dcnm/resource_dcnm_inventory.go

```diff
@@ -620,19 +677,28 @@ func resourceDCNMInventroyUpdate(d *schema.ResourceData, m interface{}) error {
 					durl := fmt.Sprintf("/rest/control/fabrics/%s/switches/%s", fabricName, serial)
 					_, delerr := dcnmClient.Delete(durl)
 					if delerr != nil {
-						log.Println("error at deletion of switch ", err)
+						diags = append(diags, diag.Diagnostic{
+							Severity: diag.Warning,
+							Detail:   fmt.Sprintf("error at deletion of switch %s", err),
+						})
 					}
 				}
 			}
-			log.Println("Some switches failed to discover and deploy, resuming procedure for successfully discovered switches")
+			diags = append(diags, diag.Diagnostic{
+				Severity: diag.Warning,
+				Detail:   "Some switches failed to discover and deploy, resuming procedure for successfully discovered switches",
```

Comment:
probably should be "some" based on other changes.

Reply:
Done